torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -179,7 +179,7 @@ def get_mutable_arg_names(op: torch._ops.OpOverload) -> List[str]:
 
 
 def do_auto_functionalize(
-    op: torch._ops.OpOverload, args: Tuple[Any, ...], kwargs: Dict[str, Any]
+    op: torch._ops.OpOverload, args: Tuple[Any, ...], kwargs: Dict[str, Any], mode
```

Comment:
auto_functionalize doesn't need to work with pre-dispatch tracing, right? In general custom ops cannot work with pre-dispatch tracing because we can't tell (without decomposing them) if they are going to be mutable

Reply:
Ok